tekton/ci/jobs/tekton-catlin-lint.yaml

```
@@ -40,11 +40,33 @@ spec:
       image: gcr.io/tekton-releases/dogfooding/catlin:latest
       workingDir: $(resources.inputs.source.path)
       script: |
+        set +e
         [[ ! -s $(workspaces.store-changed-files.path)/changed-files.txt ]] && {
           echo "No file change detected in task directory"
+          echo -n "" > $(workspaces.store-changed-files.path)/catlin.txt
           exit 0
         }
-        catlin validate $(cat $(workspaces.store-changed-files.path)/changed-files.txt)
+
+        # creating a file which will contain the final formatted output
+        # which needs to be added as a comment(if any)
+        echo '**Catlin Output**' >> $(workspaces.store-changed-files.path)/catlin.txt
+        echo '```' >> $(workspaces.store-changed-files.path)/catlin.txt
+
+        # performing catlin validate
+        catlin validate $(cat $(workspaces.store-changed-files.path)/changed-files.txt) | tee -a $(workspaces.store-changed-files.path)/catlin.txt
+        echo '```' >> $(workspaces.store-changed-files.path)/catlin.txt
+
+        # checking if there is any ERROR or WARN produces by catlin
```

```suggestion
        # checking if there are any ERROR or WARN messages produced by catlin
```
messages or maybe log messages